package.json

```diff
@@ -13,8 +13,8 @@
     "colors": "^1.1.2",
     "dotenv-safe": "^4.0.4",
     "dugite": "^1.45.0",
-    "electron-docs-linter": "^2.4.0",
-    "electron-typescript-definitions": "^7.0.0",
+    "electron-docs-linter": "^3.0.1",
+    "electron-typescript-definitions": "^8.2.2",
```

Comment:
This is not safe

This completely changes how events are documented and thus would break the generated definitions

Reply:
any suggestions how to do it in a safe way?